test/python/common.py

```diff
@@ -32,6 +33,13 @@
 class NgraphTest(object):
 
     def with_ngraph(self, l, config=tf.ConfigProto()):
+        rewrite_options = rewriter_config_pb2.RewriterConfig(custom_optimizers=[
```

Comment:
in pytests we want to request `ngraph-optimizer` only when we have compiled using grappler right? would this work with the normal build?

Reply:
This file got uploaded by mistake. I will remove it.